OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -52,147 +52,604 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            </item>
+                                            <item xml:id="q11" n="11">
+                                                <dim unit="leaf">4</dim>
+                                                <locus from="63r" to="67v"/>
+                                            </item>
+                                            <item xml:id="q12" n="12">
+                                                <dim unit="leaf">9</dim>
+                                                <locus from="68r" to="76v"/>
+                                            </item>
+                                            <item xml:id="q13" n="13">
+                                                <dim unit="leaf">8</dim>
+                                                <locus from="77r" to="84v"/>
+                                            </item>
+                                            <item xml:id="q14" n="14">
+                                                <dim unit="leaf">8</dim>
+                                                <locus from="85r" to="92v"/>
+                                                3, stub after 6
+                                                6, stub after 3
+                                            </item>
+                                            <item xml:id="q15" n="15">
+                                                <dim unit="leaf">9</dim>
+                                                <locus from="93r" to="101v"/>
+                                            </item>
+                                            <item xml:id="q16" n="16">
+                                                <dim unit="leaf">7</dim>
+                                                <locus from="102r" to="108v"/>
+                                            </item>
+                                            <item xml:id="q17" n="17">
+                                                <dim unit="leaf">6</dim>
+                                                <locus from="109r" to="114v"/>
+                                                stub between <locus target="#113"/> and <locus target="#114"/>
+                                            </item>
+                                            <item xml:id="q18" n="18">
+                                                <dim unit="leaf">9</dim>
+                                                <locus from="115r" to="123v"/>
+                                            </item>
+                                        </list>
+                                    </collation>
+                                    <condition key="deficient">
+                                        The textblock has been resewn together and parchment guards have been placed around all quires.
+                                        Leaves <locus from="51r" to="62v"/> have been stitched together at a later stage.
```

Comment:
Depending on the visualisation rules we will set, it might be better to delete "Leaves", or the result could be something like "Leaves ff. 51r..."